src/Stream/src/BinderRabbitMQ/Startup.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Steeltoe.Messaging.RabbitMQ.Config;
+using Steeltoe.Messaging.RabbitMQ.Connection;
+using Steeltoe.Stream.Attributes;
+using Steeltoe.Stream.Binder.Rabbit;
+using Steeltoe.Stream.Binder.Rabbit.Config;
+using Steeltoe.Stream.Binder.Rabbit.Provisioning;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+[assembly: Binder("rabbit", typeof(Startup))]
+
+namespace Steeltoe.Stream.Binder.Rabbit
+{
+    public class Startup
```

Comment:
Is this file supposed to be included? If yes, use of `localhost` on L37 seems odd

Reply:
fixed.